### PR TITLE
Prove revwlk

### PR DESCRIPTION
**Pull request by BTernaryTau**
> The reverse of a walk is a walk.

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -500210,6 +500210,53 @@ have become an indirect lemma of the theorem in question (i.e. a lemma
       DUGUEBAIZMUIBEAPUJUFUEBAQRSTUAABCDEFGUBUC $.
   $}
 
+  ${
+    $d I x $.  $d P k x $.  $d F k x $.  $d G k x $.
+    revwlk.1 $e |- V = ( Vtx ` G ) $.
+    revwlk.2 $e |- I = ( iEdg ` G ) $.
```
Comment (avekens) [inline set.mm:500216]: These hypothesies are not necessary, because V and I do not appear in the statement.

Comment (avekens) [reply set.mm:500216]: Therefore, they sould be removed (`V` and `I` replaced by ``( Vtx ` G )`` and ``( iEdg ` G )`` in the proof).

Comment (BTernaryTau) [reply set.mm:500216]: Done, thanks!

**Review by tirix — APPROVED** [review]

**Review by avekens — APPROVED** [review]
Using ~revrev, it should be possible to show the equivalence

`` revwlkb $p |- ( F ( Walks ` G ) P <-> ( reverse ` F ) ( Walks ` G ) ( reverse ` P ) ) ``

But this can be done in a subsequent PR.

**Pull request merged**